### DISCUSSION TITLE
Switch from pro plan to free plan

### DISCUSSION THREAD

**Discussion by Rustem555** [supabase/supabase#26908] [General]
I used the app on the free plan. I reached my limit and switched to the pro plan. I've made improvements to the app to minimize database access and would like to switch back to the free plan. However, in the mail I see this message: «Reaching out again regarding your recent invoice for your org "rakhmerov74@gmail.com's Org". Our record indicates that this invoice remains unpaid and has now passed the due date. Please let me know if you would like to keep the Pro Plan, otherwise, we will pause your project(s) in a week’s time.»
How can I now return to the free plan without pausing the project?

**Comment by GaryAustin1** [comment]
I would think you would do it here (I don't have pro plan)
![image](https://github.com/supabase/supabase/assets/54564956/cb9f1909-d4a7-4226-a9fc-cc2533d2ad07)


But normally you still pay (get a credit) for the full month.   If you upgraded to pro and did not pay for it, not sure if that could impact things.

Sounds like you should contact support if the infrastructure org option is not working for you.
